canonical/ubuntu.com
# WD-10547 Remove raspberry pi pages

## Done

Remove raspberry-pi pages [according to request](https://warthogs.atlassian.net/browse/WD-10547)

## QA

- Check following pages are removed and redirecting to https://ubuntu-com-13775.demos.haus/download/raspberry-pi:

https://ubuntu-com-13775.demos.haus/raspberry-pi 
https://ubuntu-com-13775.demos.haus/raspberry-pi/desktop 
https://ubuntu-com-13775.demos.haus/raspberry-pi/server

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10547

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
